keygen/src/keygen.rs

```diff
@@ -136,13 +136,13 @@ fn main() -> Result<(), Box<dyn error::Error>> {
             if outfile == "-" {
                 println!("{}", serialized_keypair);
             } else {
-                println!("Wrote new keypair to {}", outfile);
+                eprintln!("Wrote new keypair to {}", outfile);
             }
 
             let silent = matches.is_present("silent");
             if !silent {
                 let divider = String::from_utf8(vec![b'='; phrase.len()]).unwrap();
-                println!(
+                eprintln!(
                     "{}\nSave this mnemonic phrase to recover your new keypair:\n{}\n{}",
                     &divider, phrase, &divider
                 );
```

Comment:
In "recover" below, can you update `prompt_password_stdout` to `prompt_password_stderr`?
This allows this to work:
`$ solana-keygen recover -o - | gpg -c -o key.json.gpg`